financial-services-accelerator/components/org.wso2.financial.services.accelerator.consent.mgt.extensions/src/main/java/org/wso2/financial/services/accelerator/consent/mgt/extensions/authservlet/impl/BFSIDefaultAuthServletImpl.java

```diff
@@ -0,0 +1,96 @@
+ * <p>
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.financial.services.accelerator.consent.mgt.extensions.authservlet.impl;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import org.json.JSONObject;
+import org.wso2.financial.services.accelerator.consent.mgt.extensions.authservlet.BFSIAuthServletInterface;
+import org.wso2.financial.services.accelerator.consent.mgt.extensions.authservlet.utils.Utils;
+import org.wso2.financial.services.accelerator.consent.mgt.extensions.common.ConsentExtensionConstants;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.ResourceBundle;
+
+import javax.servlet.http.HttpServletRequest;
+
+/**
+ * The default implementation for BFSI flow.
+ */
+public class BFSIDefaultAuthServletImpl implements BFSIAuthServletInterface {
+
+    private String jspPath;
+    @Override
+    public Map<String, Object> updateRequestAttribute(HttpServletRequest request, JSONObject dataSet,
+                                                      ResourceBundle resourceBundle) {
+
+        String consentType = dataSet.getString("type");
+        switch (consentType) {
+
+            case ConsentExtensionConstants.ACCOUNTS:
+                return Utils.populateAccountsData(request, dataSet);
+            case ConsentExtensionConstants.PAYMENTS:
+                return Utils.populatePaymentsData(request, dataSet);
+            case ConsentExtensionConstants.FUNDS_CONFIRMATIONS:
+                return Utils.populateCoFData(request, dataSet);
+            default:
+                return new HashMap<>();
+        }
+    }
+
+    @Override
+    public Map<String, Object> updateSessionAttribute(HttpServletRequest request, JSONObject dataSet,
+                                                      ResourceBundle resourceBundle) {
+
+        return new HashMap<>();
+    }
+
+    @Override
+    @SuppressFBWarnings("SERVLET_PARAMETER")
+    // Suppressed content - request.getParameter("accounts[]")
+    // Suppression reason - False Positive : These endpoints are secured with access control
+    // as defined in the IS deployment.toml file
+    // Suppressed warning count - 1
+    public Map<String, Object> updateConsentData(HttpServletRequest request) {
+
+        Map<String, Object> returnMaps = new HashMap<>();
+
+        String[] accounts = request.getParameter("accounts[]").replaceAll("[\r\n]", "").split(":");
+        returnMaps.put("accountIds", List.of(accounts));
+
+        returnMaps.put(ConsentExtensionConstants.PAYMENT_ACCOUNT,
+                request.getParameter(ConsentExtensionConstants.PAYMENT_ACCOUNT).replaceAll("[\r\n]", ""));
```

Comment:
Shall we move `"[\r\n]"` to a constant? and consider using StringUtils.empty() where applicable.

Reply:
Fixed in https://github.com/wso2/financial-services-accelerator/pull/109/commits/c8dc488ccfe9eb51e8e5ed3fc3b63013f10843d0